src/PieChart.js

```diff
@@ -286,7 +293,7 @@ class PieChart extends React.Component {
       markerLineOverhangOuter,
     } = this.props;
 
-    const valueAccessor = makeAccessor(this.props.getValue);
+    const valueAccessor = makeAccessor(this.props.slice);
```

Comment:
Let's de-structure slice from line 294

Reply:
Slice is used a variable in multiple instances throughout the code. There are scoping conflicts when this is adjusted. I think we can stick with `this.props.slice` for now.